components/common/src/main/java/com/hotels/styx/config/schema/PassValidation.java

```diff
@@ -0,0 +1,22 @@
+/*
+  Copyright (C) 2013-2018 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx.config.schema;
+
+/**
+ * A directive that marks the object as opaque.
```

Comment:
A note about naming. I meant to rename this to `OpaqueSchema` or perhaps `UnknownSchema`. This is because `Pass` is a verb, indicating an action taking place. As such it feels out of place in a declarative DSL. However `OpaqueSchema` is more descriptive and fits better in the DSL.